ecl/eclcc/eclcc.cpp

```diff
@@ -870,6 +867,24 @@ void gatherResourceManifestFilenames(EclCompileInstance & instance, StringArray
                 filenames.append(filename);
         }
     }
+
+}
+
+void gatherResourceManifestFilenames(EclCompileInstance & instance, StringArray &filenames)
+{
+    if (instance.archive)
+    {
+        gatherResourceManifestFilenames(instance.archive, "Module/Attribute", filenames);
+
+        //Gather resources from dependent packages
```

Comment:
This is the main bug fix for the jira - the new git dependencies have nested \<Archive>s with the main Archive